sdk/openai/azure-openai/samples/chat_completions_oyd.py

```diff
@@ -0,0 +1,53 @@
+def chat_completion_oyd_studio_viewcode() -> None:
+
+    import os
+    from openai import AzureOpenAI
+    from azure.identity import DefaultAzureCredential, get_bearer_token_provider
+
+    endpoint = os.environ["AZURE_OPENAI_ENDPOINT"]
+    deployment = os.environ["AZURE_OPENAI_CHAT_DEPLOYMENT"]
+
+    #token_provider = get_bearer_token_provider(DefaultAzureCredential(), "https://cognitiveservices.azure.com/.default")
+
+    client = AzureOpenAI(
+        azure_endpoint = endpoint,
+        api_key = os.environ["AZURE_OPENAI_KEY"],
+        api_version = "2024-02-01"
```

Comment:
nit on the spacing here. Recommend running `black` on the files to standardize formatting. 

pip install black
black chat_completions_oyd.py